### PR TITLE
New package: RadioDJ.RadioDJ version 2.0.4.3

### LINKED ISSUE #132409: [Package Request]: RadioDJ
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://www.radiodj.ro/download
Publisher: radiodj.ro
Package Name: RadioDJ
Description: Radio Automation Software.
Package Version: 2.0.4.3
Installer URL: https://www.radiodj.ro/download/latest_version/SetupRadioDJ2043Free.zip

### PR DESCRIPTION
**Pull request by Exorcism0666**
### Pull request has been created with [Komac](https://github.com/russellbanks/Komac) v1.11.0 :rocket:

- Resolve: #132409 

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/132412)

### CODE REVIEW

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping @Exorcism0666, I'll get a notification.
>[!important]
>Please recheck these Pull Request carefully before merging. If it is a incorrect-removing Pull Request and the URLs don't have any issue while manual checking, please close this Pull Request directly(It's best to let Exorcism0666 know).

良い一日をお過ごしください！

File: manifests/r/RadioDJ/RadioDJ/2.0.4.3/RadioDJ.RadioDJ.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: RadioDJ.RadioDJ
+PackageVersion: 2.0.4.3
+InstallerType: zip
+NestedInstallerType: exe
```
Comment (Trenly) [inline manifests/r/RadioDJ/RadioDJ/2.0.4.3/RadioDJ.RadioDJ.installer.yaml:7]:
```suggestion
NestedInstallerType: inno
```

File: manifests/r/RadioDJ/RadioDJ/2.0.4.3/RadioDJ.RadioDJ.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: RadioDJ.RadioDJ
+PackageVersion: 2.0.4.3
+PackageLocale: en-US
+Publisher: RadioDJ.ro
```
Comment (Trenly) [inline manifests/r/RadioDJ/RadioDJ/2.0.4.3/RadioDJ.RadioDJ.locale.en-US.yaml:7]:
```suggestion
Publisher: Broadcast Mediaware, LLC
```
This must match control Panel

File: manifests/r/RadioDJ/RadioDJ/2.0.4.3/RadioDJ.RadioDJ.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: RadioDJ.RadioDJ
+PackageVersion: 2.0.4.3
+PackageLocale: en-US
+Publisher: RadioDJ.ro
+PublisherSupportUrl: https://radiodj.ro/#
+PackageName: RadioDJ
+License: Proprietary
```
Comment (Trenly) [inline manifests/r/RadioDJ/RadioDJ/2.0.4.3/RadioDJ.RadioDJ.locale.en-US.yaml:10]:
```suggestion
License: Proprietary
LicenseUrl: https://www.radiodj.ro/license
```

**Review by Trenly — CHANGES_REQUESTED** [review]
Needs `InstallerSwitches` added for a silent install -
image

**Comment by stephengillie** [comment]
> Needs `InstallerSwitches` added for a silent install - image

This is unchanged.

**Comment by Exorcism0666** [comment]
image

File: manifests/r/RadioDJ/RadioDJ/2.0.4.3/RadioDJ.RadioDJ.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: RadioDJ.RadioDJ
+PackageVersion: 2.0.4.3
+InstallerType: zip
+InstallerSwitches:
+  Silent: /VERYSILENT
+  SilentWithProgress: /SILENT
+  Custom: ''
```
Comment (OfficialEsco) [inline manifests/r/RadioDJ/RadioDJ/2.0.4.3/RadioDJ.RadioDJ.installer.yaml:10]:
```suggestion
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

**Review by OfficialEsco — COMMENTED** [review]
wops
image
ill take a look

**Review by OfficialEsco — COMMENTED** [review]
I think this is a issue with the installer it self, /SILENT and /VERYSILENT does not take affect before you click Yes/No, /SUPRESSMSGBOXES does also not work since you're not actually at the Inno installer

Edit:
Found it, you have to use /SP-

File: manifests/r/RadioDJ/RadioDJ/2.0.4.3/RadioDJ.RadioDJ.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,16 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: RadioDJ.RadioDJ
+PackageVersion: 2.0.4.3
+InstallerType: zip
```
Comment (OfficialEsco) [inline manifests/r/RadioDJ/RadioDJ/2.0.4.3/RadioDJ.RadioDJ.installer.yaml:6]:
```suggestion
InstallerType: zip
InstallerSwitches:
  Custom: '/SP-'
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

**Comment by OfficialEsco** [comment]
It does however stay stuck on "Starting package install.." until you close the app..
image

**Comment by stephengillie** [comment]
What we're seeing is the Database Setup utility. Running the attended installer gives a checkbox: 

image

Is there an installer switch to skip this step? 

Alternatively, is there a switch to automate setting it up? Closing it without setting up a database, and opening the application, gives this message: 
image

What is the recommended way to set up this application?

**Pull request closed without merging**